metabase/metabase
# Mantine Modal

Epic #32268

Ports [Mantine Modal](https://v6.mantine.dev/core/modal/) to `metabase/ui/overlays`

### To verify

1. `yarn storybook`
2. Open `https://localhost:6006/?path=/story/overlays-modal`
3. Ensure the modal looks and works as expected
